src/olympia/abuse/models.py

```diff
@@ -1044,7 +1045,11 @@ def notify_reviewer_decision(
         )
         # override target_url if this decision related to unlisted versions
         target_url_override = (
-            {'target_url': reverse('devhub.addons.versions', args=[self.target.id])}
+            {
```

Comment:
What is the use case for not absolutifying a URL? Like when would we want just /banana/foo ? Is there some benefit to that sometime?

Reply:
In emails, there isn't a use-case.  In page templates we normally don't want absolute urls.